test/express/handlers.js

```diff
@@ -0,0 +1,165 @@
+            );
+            assert.strictEqual(errorResult.statusCode, "2000");
+            assert.strictEqual(errorResult.code, "2000");
+            assert.strictEqual(errorResult.error, "Error");
+            assert.deepStrictEqual(errorResult.stack, ["Error", "Error"]);
+
+            ripeCommons.errorHandlerExpress(
+                {
+                    name: "Error",
+                    code: "404",
+                    message: "Error 404"
+                },
+                {},
+                res,
+                next
+            );
+            assert.strictEqual(errorResult.statusCode, "404");
+            assert.strictEqual(errorResult.code, "404");
+            assert.strictEqual(errorResult.error, "Error 404");
+            assert.deepStrictEqual(errorResult.stack, []);
+        });
+
+        it("should default to error code 500 if none is provided", () => {
+            let errorResult = {};
+            const res = {
+                status: statusNumber => {
+                    errorResult = { statusCode: statusNumber };
+                },
+                json: payload => {
+                    errorResult = { ...errorResult, ...payload };
+                }
+            };
+
+            ripeCommons.errorHandlerExpress(
+                {
+                    name: "Error",
+                    message: "Error",
+                    stack: "Error\nError"
+                },
+                {},
+                res
+            );
+            assert.strictEqual(errorResult.statusCode, 500);
+            assert.strictEqual(errorResult.code, 500);
+            assert.strictEqual(errorResult.error, "Error");
+            assert.deepStrictEqual(errorResult.stack, ["Error", "Error"]);
+        });
+
+        it("should not populate the stack trace field in production environment", () => {
+            let errorResult = {};
+            const res = {
+                status: statusNumber => {
+                    errorResult = { statusCode: statusNumber };
+                },
+                json: payload => {
+                    errorResult = { ...errorResult, ...payload };
+                }
+            };
+            const next = () => {};
+            process.env.NODE_ENV = "production";
+
+            try {
+                ripeCommons.errorHandlerExpress(
+                    {
+                        name: "Error",
+                        code: "400",
+                        message: "Error",
+                        stack: "Error\nError"
+                    },
+                    {},
+                    res,
+                    next
+                );
+                assert.strictEqual(errorResult.statusCode, "400");
+                assert.strictEqual(errorResult.code, "400");
+                assert.strictEqual(errorResult.error, "Error");
+                assert.strictEqual(errorResult.stack, undefined);
+            } finally {
+                process.env.NODE_ENV = "development";

```

Comment:
Should we assume the previous was development? Maybe save the previous value like

```js
const prev = process.env.NODE_ENV;
process.env.NODE_ENV = "production";
....
finally {
  process.env.NODE_ENV = prev;
}
```